sequencing/vcf_to_plink/dxapp.json

```diff
@@ -10,7 +10,7 @@
   ],
   "inputSpec": [
     {
-      "name": "vcf_fn",
+      "name": "variants_vcfgz",
```

Comment:
Should this name end with an s for "array" of files?
"variants_vcfgz" -> "variants_vcfgzs"
I'll leave this for you to make a final decision here.